modules.d/06rngd/module-setup.sh

```diff
@@ -33,6 +33,8 @@ check() {
 install() {
     inst rngd
     inst_simple "${moddir}/rngd.service" "${systemdsystemunitdir}/rngd.service"
+    # make sure dependant libs are installed too
+    inst_libdir_file opensc-pkcs11.so
```

Comment:
This is not a hard dependency on all platforms, e.g. openSUSE, so now, this line will fail there.

Reply:
start rngd with the -x pkcs11 option, that should prevent that entropy source from initializing, and  squash the library dependency, so it need not be included in the initramfs